modules/javascript/src/main/webapp/javascript/atmosphere.js

```diff
@@ -2504,7 +2511,7 @@
                                         res.innerText = "";
                                         var skipCallbackInvocation = _trackMessageSize(text, rq, _response);
                                         if (skipCallbackInvocation) {
-                                            return "";
+                                            return false;
```

Comment:
Should be equivalent at run time, but it is more clear if the return type is the same across the function.